modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/view/table/SystemRelatedModelsTableFDSView.java

```diff
@@ -168,7 +168,7 @@ public List<RelatedModel> getItems(
 					GetterUtil.getLong(
 						modelAttributes.get(
 							objectDefinition.getPKObjectFieldDBColumnName())),
-					value.toString(), true);
+					String.valueOf(value), true);
```

Comment:
Did you try to use the GetterUtil.getString()? Could you test it, please? Maybe, it could be safer.

Reply:
Hi  @marcelabc 
I just tested it. When the GetterUtil.getString() is called with a null value, it returns an empty string. Whereas the String.valueOf() returns "null" as a string. Below are the two results. Which one should we go with? 

![withValueOf](https://user-images.githubusercontent.com/82452840/212381623-5857300d-35f6-4851-9a1c-9df2395add52.png)
![withGetterUtil](https://user-images.githubusercontent.com/82452840/212381628-9262bb20-17a0-4197-897d-1e883a2b5720.png)

Reply:
Despite the situation and the way to fix be very close in both this and the another one described in [LPS-171844](https://issues.liferay.com/browse/LPS-171844) , however, the fix of these tickets behavior are independent. Fixing one do not fix the another.